### PR TITLE
add task popup DmitryHniezdilov

### PR DESCRIPTION
**Pull request by DmitryHniezdilov**
### HTML-CSS-Popup

[Demo](https://dmitryhniezdilov.github.io/task-popup/) |
[Code base](https://github.com/DmitryHniezdilov/task-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by zonzujiro** [comment]
You should have a possibility to open menu with Space button!

**Comment by DmitryHniezdilov** [comment]
Hi. Sorry for that this simple bug, I never fixed this bug before, and check only using tab.

Please, review.

File: submissions/DmitryHniezdilov/task-html-css-popup/css/style.css [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+.header__nav-icon {
+  width: 25px;
+  height: 25px;
+}
+
+.popup {
+  position: relative;
+}
+
+.popup__input,
```
Comment (A-Ostrovnyy) [inline submissions/DmitryHniezdilov/task-html-css-popup/css/style.css:110]: Watch [the video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) to find out how to hide the checkbox in the right way.

File: submissions/DmitryHniezdilov/task-html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+                <nav class="header__nav">
+                    <ul class="header__nav-list">
+                        <li class="header__nav-item">
+                            <a href="#" class="header__nav-link" tabindex="1">Mail</a>
+                        </li>
+                        <li class="header__nav-item">
+                            <a href="#" class="header__nav-link" tabindex="1">Images</a>
+                        </li>
+                        <li class="header__nav-item header__nav-item--popup popup">
+                            <input class="popup__input" id="popup" type="checkbox" tabindex="1">
```
Comment (A-Ostrovnyy) [inline submissions/DmitryHniezdilov/task-html-css-popup/index.html:23]: Your markup will work fine without `tabindex` attribute.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

**Comment by DmitryHniezdilov** [comment]
Thanks for the review. It will help me to check the code more carefully in the future.

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Comment by DmitryHniezdilov** [comment]
Thank you!

**Pull request merged**